cmd/spire-server/cli/entry/show.go

```diff
@@ -15,6 +15,8 @@ import (
 	"golang.org/x/net/context"
 )
 
+const listEntriesRequestPageSize = 1000
```

Comment:
I do not understand what size should be defined...

Reply:
Fix: https://github.com/spiffe/spire/pull/3135/commits/ee8ea5dea0ffa8e9d4f1d820e1b2b40adcd26578